manage

```diff
@@ -101,6 +101,43 @@ usage () {
         Example:
         $0 generategenesisfiles "./tmp/CANdy Beta Genesis File Node info - Trustees.csv" "./tmp/CANdy Beta Genesis File Node info - Stewards.csv"
 
+  apply-taa - Registers an AML and TAA on a given network.
+              This is a convent wrapper around the cli-scripts/apply-taa batch script,
+              which simplifies and automates the process of registering an AML and TAA.
+
+              The following parameters are simply the standard inputs for the indy-cli batch script.
+              Refer to the cli-scripts/apply-taa documentation for details;
+              - walletName          - Required
+              - storageType         - Optional
+              - storageConfig       - Optional
+              - storageCredentials  - Optional
+              - poolName            - Required
+              - useDid              - Required
+
+              The following parameters are required:
+              amlUrl
+                - The URL to the raw content of the AML.  Provided the link contains the AML expected content in json format 
+                  (contained in {}s) it will be parsed from the link's text.
+              amlVersion
+                - The version of the AML.
+              taaUrl
+                - The URL to the raw content of the TAA in markdown format.
+              taaVersion
+                - The vesion of the TAA
+              taaRatificationTime
+                - The date and time of TAA ratification by network government.
+
+        Example:
```

Comment:
Should this example work?  When I try (against a local von-network) I get:

```
$ ./manage apply-taa                 walletName=local_net_trustee_wallet                 poolName=local_net                 useDid=V4SGRU86Z58d6TV7PBUe6f                 amlUrl='https://raw.githubusercontent.com/wiki/bcgov/bc-vcpedia/(Layer-1)-CANdy-Acceptance-Mechanism-List-(AML).md'                 amlVersion=0.1                 taaUrl='https://raw.githubusercontent.com/wiki/bcgov/bc-vcpedia/(Layer-1)-CANdy-Transaction-Author-Agreement.md'                 taaVersion=0.1                 taaRatificationTime="2022-08-09T11:05:00-0700"

Using: docker-compose --log-level ERROR

date: illegal option -- -
usage: date [-jnRu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ... 
            [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]
sed: 1: "/{/,/}/{/{/!{/}/!p}}": extra characters at the end of p command
Creating von_client_run ... done
Can't read the file
Goodbye...
```

Reply:
Yes, it should work.  That is the command I was using for testing.

Reply:
Do you get the same error if you run this command:
```
date --date="2022-08-09T11:05:00-0700" +"%s"
```

Reply:
> Do you get the same error if you run this command?
> 
> ```
> date --date="2022-08-09T11:05:00-0700" +"%s"
> ```
> 
> The expected result is `1660068300`

Yes same error, I'll try your mac version

Reply:
Should be fixed in the most recent commit.